Sample_v3/AppDelegate.swift

```diff
@@ -47,4 +51,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
         // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
     }
+    
+    private func setupAppearance() {
+        ChatChannelListCollectionView.appearance().backgroundColor = .white
+        AvatarView.appearance().backgroundColor = .gray
+    }
```

Comment:
This is what the default appearance will be, right?

Reply:
Yes, correct